File: mambo-protocol/src/main/scala/org/mambo/protocol/enums/TaxCollectorErrorReasonEnum.scala




















// Generated on 05/02/2013 20:27:17
// Scala version by Blackrush
package org.mambo.protocol.enums

import org.mambo.protocol.Enumeration

object TaxCollectorErrorReasonEnum extends Enumeration {
  val taxCollectorErrorUnknown = Value(0, "taxCollectorErrorUnknown")
  val taxCollectorNotFound = Value(1, "taxCollectorNotFound")
  val taxCollectorNotOwned = Value(2, "taxCollectorNotOwned")
  val taxCollectorNoRights = Value(3, "taxCollectorNoRights")
  val taxCollectorMaxReached = Value(4, "taxCollectorMaxReached")
  val taxCollectorAlreadyOne = Value(5, "taxCollectorAlreadyOne")
  val taxCollectorCantHireYet = Value(6, "taxCollectorCantHireYet")
  val taxCollectorCantHireHere = Value(7, "taxCollectorCantHireHere")
  val taxCollectorNotEnoughKamas = Value(8, "taxCollectorNotEnoughKamas")
}

File: mambo-protocol/src/main/java/org/mambo/protocol/enums/TaxCollectorErrorReasonEnum.java


// Generated on 05/08/2013 19:04:45
package org.mambo.protocol.enums;

public enum TaxCollectorErrorReasonEnum {
    TAX_COLLECTOR_ERROR_UNKNOWN(0),
    TAX_COLLECTOR_NOT_FOUND(1),
    TAX_COLLECTOR_NOT_OWNED(2),
    TAX_COLLECTOR_NO_RIGHTS(3),
    TAX_COLLECTOR_MAX_REACHED(4),
    TAX_COLLECTOR_ALREADY_ONE(5),
    TAX_COLLECTOR_CANT_HIRE_YET(6),
    TAX_COLLECTOR_CANT_HIRE_HERE(7),
    TAX_COLLECTOR_NOT_ENOUGH_KAMAS(8);
    
    private int value;
    TaxCollectorErrorReasonEnum(int value) { this.value = value; }
    public int value() { return value; }
    
    public static TaxCollectorErrorReasonEnum valueOf(int value) {
    	for (TaxCollectorErrorReasonEnum a : values()) {
    		if (a.value == value)
    			return a;
    	}
    	return null;
    }
    
}
